.travis.yml

```diff
@@ -9,13 +7,16 @@ env:
   - DJANGO_SETTINGS_MODULE="settings_test"
 
 python:
-  - "2.7"
-  - "3.4"
   - "3.5"
   - "3.6"
   - "3.7"
   - "3.8"
 
+matrix:
+  include:
+    - env: TOXENV=lint
+      python: 3.8
```

Comment:
The Python version is specified explicitly here 'cause it was running at 3.5 by default, which does not support f-strings, which we've patched over by installing `future-fstrings`. However `skip_install` in the `tox` config is enabled for this linting environment so we weren't actually installing it.

Reply:
Yup, the 3.5 build installs `future-fstrings` as part of its [standard installation requirements](https://github.com/azavea/django-ecsmanage/blob/08b0bd72bbd212765960cd2e806ac2b00988979f/setup.cfg#L29-L32). So that's getting installed when we run unit tests for 3.5 with `tox` but not when we run linting.

`django-amazon-ses` is running linting with 3.5 (I think it defaults to whichever version is declared first in the `python` list in `travis.yml`) and [passing](https://travis-ci.org/azavea/django-amazon-ses/jobs/648061400) because it isn't using f-strings.

The `lint` `tox` environment was [failing](https://travis-ci.org/azavea/django-ecsmanage/jobs/655577224) when running on 3.5 and [passes](https://travis-ci.org/azavea/django-ecsmanage/jobs/657367558) on 3.8. We could install `future-fstrings` in the `tox` dependencies but I think it's preferable for us to lint with the most recent stable version of Python. That way we can use more recent versions of our linters and tools that might not support older versions of Python.

If anything I think we should update `django-amazon-ses` to explicitly use 3.8 unless we have made a conscious decision to use 3.5.

Reply:
If we're supporting 3.5 as the oldest version, I have a desire to make sure all of our checks pass on 3.5+. With that logic, what do you think of running linting across all versions? Can we install `future-fstring` and invoke linting in each build?

Reply:
I was basing these changes off of azavea/django-amazon-ses#61 which was introduced to stop linting across all versions. The rationale for that PR was that linting only needs to be run once for the whole project and we don't need to lint for each version. This speeds up the rest of the build by removing the need to lint on each variant build.

My observation was that the failure on 3.5 wasn't indicating a linting error in the source but a parsing problem with the linter's runtime.

Reply:
There are two rationales in my head:

- Support 3.8 so we don't have to install `future-fstrings`
- Support 3.5 so we know there is compatibility with the oldest, supported version

I don't feel strongly about either of them. I'm fine with you forging the path from here.

Reply:
I thought about this pretty hard, and I'm in favor of running our linter on 3.5.

We can catch syntax errors this way, but it feels kinda like blanket code coverage and something that we should be catching with automated tests. However we don't have code coverage on those paths right now and we'd need to prioritize increasing coverage.

Practicality beats principle for me so I'd rather we rely on a blanket approach than potentially miss errors that break backwards compatibility. I've confirmed that the install / test process would totally miss if we forgot to install `future-fstrings` and that's reason enough for me.